include/deal.II/lac/affine_constraints.h

```diff
@@ -770,6 +770,22 @@ class AffineConstraints : public Subscriptor
   void
   close();
 
+  /**
+   * Check if the function close() was called or there are no
+   * constraints locally, which is normally the case if a dummy
+   * AffineConstraints was created for the DG case.
+   */
+  bool
+  is_closed() const;
+
+  /**
+   * Check if the function close() was called or there are no
+   * constraints globally, which is normally the case if a dummy
+   * AffineConstraints was created for the DG case.
+   */
+  bool
+  is_closed(const MPI_Comm &comm) const;
```

Comment:
Do you think it would make sense to select a more descriptive name here?
```suggestion
  is_closed_on_all_processes(const MPI_Comm &comm) const;
```
(Given that the `AffineConstraints` class does not have the notion of the parallel distribution built-in.)

Reply:
To be consistent would be `is_closed_in_parallel` (similar to `is_consistent_in_parallel`). The reason why I opted against this is that normally one calls or not `close()` on all processes (the only reason for having the reduction inisde to be able to decide if the index set is empty - since the constructor sets `sorted` automatically to false). If you think it is more clear, I can rename the function to `is_closed_in_parallel`.